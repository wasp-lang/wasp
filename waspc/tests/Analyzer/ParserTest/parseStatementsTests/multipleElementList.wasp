test Decl [ 1, 2, 3 ]
