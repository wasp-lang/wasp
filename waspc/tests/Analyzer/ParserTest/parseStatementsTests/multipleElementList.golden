(AST
  (Decl@1:1-21 type=test name=Decl
    (List@1:11-21
      (Integer@1:13 value=1)
      (Integer@1:16 value=2)
      (Integer@1:19 value=3)
    )
  )
)
